backend/src/main/java/com/jobwait/persistence/PostgresController.java

```diff
@@ -133,12 +134,12 @@ INSERT INTO answers (%s)
     }
 
     @Override
-    public User createUserFromAuthId(String authHash) {
+    public User createUser(Tuple<String, String> userIDAndHash) {
         try {
             Connection connection = getConnection();
 
             PreparedStatement statement = connection
-                    .prepareStatement("INSERT INTO users(authhash) VALUES (?) RETURNING *");
+                    .prepareStatement("INSERT INTO users(id, authhash) VALUES (?, ?) RETURNING *");
```

Comment:
Don't forget to update `schema.sql` to remove the auto generation of a user UUID if none is provided (in the table column definition). We want it to fail if for whatever reason it ends up blank in the SQL statement.

Reply:
Also note that if you update `schema.sql`, it won't actually update the database because we have a different sql file in the postgresql folder now. You'd have to swap those to test this properly. I guess we would also then have to perform SQL commands on the prod server to update the schema, but I can take care of that